### PR TITLE
docs: add documentation to create did:web

### PR DESCRIPTION
**Pull request by hoa-ngo-gs**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

Add crash course about Decentralised Identifiers (DIDs), how they enable self-sovereign digital identities, and how to create, manage, and host a DID:web identifier using VCkit.

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 [vc-kit doc site](https://uncefact.github.io/vckit/)
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: documentation/docs/get-started/did-web/how-to-create/hosting-did-web.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+- A registered domain (e.g., `example.com`).
+- The ability to host files on that domain.
+- HTTPS enabled on your domain, typically via SSL certificates (e.g., via Let's Encrypt or other providers).
+
+Once your domain is set up with HTTPS, you can proceed to create and host your `did:web` identifier.
+
+#### Alternative: Testing with Ngrok (Without a Domain)
+
+If you don't have a domain but want to test the process locally, you can use **ngrok** to create a secure HTTPS tunnel to your localhost. Ngrok temporarily exposes your local server to the internet, giving you a unique URL for testing.
```
Comment (huynguyen-hl) [inline documentation/docs/get-started/did-web/how-to-create/hosting-did-web.md:28]: I think the word **ngrok** could be turned into a hyperlink, so users can click on it and be redirected to the https://ngrok.com site.

File: documentation/docs/get-started/did-web/how-to-create/hosting-did-web.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+**Steps to Use Ngrok:**
+
+1. Install and register **ngrok**.
```
Comment (huynguyen-hl) [inline documentation/docs/get-started/did-web/how-to-create/hosting-did-web.md:32]: I think the word **ngrok** could be turned into a hyperlink, so users can click on it and be redirected to the https://ngrok.com/download site.

**Review by huynguyen-hl — APPROVED** [review]

**Comment by ashleythedeveloper** [comment]
LGTM

**Review by ashleythedeveloper — APPROVED** [review]
NIT: I feel that the disclaimer might seed some confusion within the user as it states that the content is not ready for implementation but is guiding the user on how to seed the identifier.

Screenshot 2024-10-16 at 7 49 28 AM

**Review by huynguyen-hl — APPROVED** [review]

**Pull request merged**